alumet/Cargo.toml

```diff
@@ -24,6 +24,7 @@ smallvec = { version = "1.13.2", features = ["union"] }
 tokio-util = "0.7.10"
 indoc = "2.0.5"
 thiserror = "1.0.62"
+humantime-serde = "1.1.1"
```

Comment:
1. This is only used in tests, it should be a dev dependency, not a regular dependency.
2. Why do you need it? It's only useful when there is a human that writes the config, but this is an automatic test.